### PR TITLE
Remove custom test runner and fixture files

### PR DESCRIPTION
**Pull request by Turbo87**
/cc @trabus

### CODE REVIEW

**Comment by Turbo87** [comment]
@trabus r?

File: README.md [hunk 1]
```diff
@@ -32,14 +32,14 @@ Usage
 The blueprint tests can be run by:
 
 ```
-node node-tests/nodetest-runner.js
+node_modules/.bin/mocha node-tests --recursive
```
Comment (stefanpenner) [inline README.md]: isn't this just `npm run nodetest` ?

**Comment by stefanpenner** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (Turbo87) [inline README.md]: if you apply the following advice: yes

**Comment by trabus** [comment]
Sorry for the delay, LGTM, thanks for merging.

**Pull request merged**